saleor/graphql/schema.graphql

```diff
@@ -893,6 +907,7 @@ type ChannelUpdate {
 }
 
 input ChannelUpdateInput {
+  isActive: Boolean
```

Comment:
We can manage the `isActive` flag through "update" and "create" mutations, plus we have dedicated mutations for that. Is this on purpose? Do we need them in this case?

Reply:
These changes were made as intended in ticket description.